packages/generator/generators/react-cmf/templates/webpack.config.js

```diff
@@ -1,79 +1,68 @@
 const autoprefixer = require('autoprefixer');
-const webpack = require('webpack');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
-const path = require('path');
+
+const extractCSS = new ExtractTextPlugin({ filename: '[name]-[hash].css' });
 
 const SASS_DATA = `$brand-primary: #77828A;
-@import '~bootstrap-talend-theme/src/theme/guidelines';
+@import '~@talend/bootstrap-theme/src/theme/guidelines';
 `;
 
+function getCommonStyleLoaders(enableModules) {
+	const cssOptions = enableModules ?
+		{ sourceMap: true, modules: true, importLoaders: 1, localIdentName: '[name]__[local]___[hash:base64:5]' } :
+		{};
+	return [
+		{ loader: 'css-loader', options: cssOptions },
+		{ loader: 'postcss-loader', options: { sourceMap: true, plugins: () => [autoprefixer({ browsers: ['last 2 versions'] })] } },
+		{ loader: 'resolve-url-loader' },
+	];
+}
+
+function getSassLoaders(enableModules) {
+	return getCommonStyleLoaders(enableModules).concat({ loader: 'sass-loader', options: { sourceMap: true, data: SASS_DATA } });
+}
+
 module.exports = {
 	entry: ['babel-polyfill', 'whatwg-fetch', './src/app/index.js'],
 	output: {
 		path: `${__dirname}/build`,
 		publicPath: '/',
-		filename: '[hash].app.js',
+		filename: '[name]-[hash].js',
 	},
-	resolve: ['', '.scss', '.css', 'js', 'jsx'],
 	module: {
 		loaders: [
 			{
 				test: /\.js$/,
 				exclude: /node_modules/,
-				loader: 'babel-loader',
+				use: { loader: 'babel-loader' },
 			},
 			{
 				test: /\.css$/,
-				loader: ExtractTextPlugin.extract('style', 'css', {
-					publicPath: './',
-				}),
+				use: extractCSS.extract(getCommonStyleLoaders()),
+				exclude: /@talend/,
 			},
 			{
-				test: /theme.scss$/,
-				loader: ExtractTextPlugin.extract(
-					'css!postcss!sass', {
-						publicPath: './',
-					}
-				),
+				test: /\.scss$/,
+				use: extractCSS.extract(getSassLoaders()),
+				exclude: /@talend/,
 			},
+			// css modules local scope
 			{
 				test: /\.scss$/,
-				exclude: /theme.scss/,
-				loader: ExtractTextPlugin.extract(
-					'css?sourceMap&modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!sass', {
-						publicPath: './',
-					}
-				),
+				use: extractCSS.extract(getSassLoaders(true)),
+				include: /@talend/,
 			},
 			{
-				test: /\.woff(2)?(\?[a-z0-9=&.]+)?$/,
-				loader: 'url',
-				query: {
-					limit: 50000,
-					mimetype: 'application/font-woff',
-					name: './fonts/[name].[ext]',
-				},
-			}
-		],
-	},
-	sassLoader: {
-		includePaths: [
-			path.resolve(
-				__dirname,
-				'node_modules'
-			),
+				test: /\.woff(2)?(\?v=\d+\.\d+\.\d+)?$/,
+				loader: 'url-loader',
+				options: { name: './fonts/[name].[ext]', limit: 10000, mimetype: 'application/font-woff' },
```

Comment:
Where is t`10000` coming from?

Reply:
Previously it was 50000 :) What value we should take ?